source/tutorials/conda-tutorial.rst

```diff
@@ -360,7 +360,8 @@ documentation <https://conda.io/docs/user-guide/tutorials/build-pkgs-skeleton.ht
 
 .. code:: bash
 
-    anaconda upload /Users/claire/anaconda/conda-bld/noarch/q2_perc_norm-2018.4.0-py35_0.tar.bz2
+    anaconda upload /Users/claire/anaconda/envs/qiime2-2019.1/conda-bld/noarch/q2_perc_norm-2019.4-py36_0.tar.bz2
```

Comment:
```suggestion
    anaconda upload /Users/claire/anaconda/envs/qiime2-2019.1/conda-bld/noarch/q2_perc_norm-2019.1-py36_0.tar.bz2
```

Reply:
ah, yeah. the `2019.4` in that file name is actually the version of my plugin (not of qiime2). I agree it's confusing, but maybe we can change it to something different that doesn't look like a qiime2 version? like `v2` or something basic like that

Reply:
oh i like that idea. timeless!

Reply:
(v2, that is)